Scenes/MainScenes/Game.cs

```diff
@@ -9,6 +10,8 @@ public class Game : Node
   private ServerConnection serverConnection;
   private LoginScreen loginScreen;
 
+  private Stopwatch GameStopwatch = new Stopwatch();
```

Comment:
not a critique, just a Q: did you consider using Godot:Timer
https://docs.godotengine.org/en/stable/classes/class_timer.html

Reply:
I didn't consider it. Stopwatch was recommended on the Godot discord because I was just trying to barf out elapsed times when I was chasing some bizarre performance issue. For that purpose it's probably better because there's no Godot overhead.

For game timing-y stuff, yes, Timer would probably be good.